app/models/ems_refresh/save_inventory_cloud.rb

```diff
@@ -68,7 +66,6 @@ def save_ems_cloud_inventory(ems, hashes, target = nil)
     # Save and link other subsections
     save_child_inventory(ems, hashes, child_keys, target)
 
-    link_volumes_to_base_snapshots(hashes[:cloud_volumes]) if hashes.key?(:cloud_volumes)
```

Comment:
My first commit took it out and my second commit put it back.  The version on my disk contains the line.  However github shows it being deleted.  I am confused.